dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/DeepStorageBrowserPresenter.java

```diff
@@ -111,11 +112,22 @@
     @Inject
     private Workers workers;
 
+    @Inject
+    private LoggingService loggingService;
+
     private DeepStorageBrowserTaskProgressView<Ds3JobTask> jobProgressView;
 
     @Override
     public void initialize(final URL location, final ResourceBundle resources) {
         try {
+            loggingService.registerLogListener( (message, logType) -> {
```

Comment:
This should be pulled out into it's own method, and then use a method reference here to register it.